docs/getting_started/quickstart.qmd

```diff
@@ -0,0 +1,403 @@
+:::
+
+### Start the server
+
+Run the server with your build hash:
+
+::: {.panel-tabset}
+
+### macOS/Linux
+```bash
+xorq serve-unbound builds/$BUILD_HASH \
+  --host localhost \
+  --port 8001 \
+  --to_unbind_hash da8ba93cc97709f4ad9502e16d71e8e3
+```
+
+### Windows (PowerShell)
+```powershell
+xorq serve-unbound builds/$env:BUILD_HASH `
+  --host localhost `
+  --port 8001 `
+  --to_unbind_hash da8ba93cc97709f4ad9502e16d71e8e3
+```
+
+### Windows (CMD)
+```cmd
+xorq serve-unbound builds/%BUILD_HASH% --host localhost --port 8001 --to_unbind_hash da8ba93cc97709f4ad9502e16d71e8e3
+```
+
+:::
+
+The command saves prediction results to `predictions.parquet`. You can open this file with any Parquet-compatible tool. Compatible tools include pandas, DuckDB, or Polars.
+
+With your pipeline running locally, the next step is to serve a UDXF expression as an endpoint.
+
+
+## Step 5: Serve a UDXF expression
+
+::: {.callout-note}
+### New terminal setup
+Open a new terminal window. Activate the same virtual environment (`.venv`) and navigate to the same directory (`penguins_example`) where your server is running.
+:::
+
+Create a file called `query_endpoint.py`:
+
+```python
+# udxf_example.py
+import pandas as pd
+import xorq.api as xo
+
+# <1>
+def add_computed_column(df: pd.DataFrame) -> pd.DataFrame:
+    """Add a computed column that doubles the input value."""
+    result = df.copy()
+    result["doubled"] = result["value"] * 2
+    return result
+
+# <2>
+input_schema = xo.schema({"value": "int64"})
+output_schema = xo.schema({"value": "int64", "doubled": "int64"})
+
+# <3>
+con = xo.connect()
+input_table = xo.memtable({"value": [1, 2, 3, 4, 5]}, schema=input_schema)
+
+# <4>
+expr = xo.expr.relations.flight_udxf(
+    input_table,
+    process_df=add_computed_column,
+    maybe_schema_in=input_schema,
+    maybe_schema_out=output_schema,
+    con=con,
+    make_udxf_kwargs={
+        "name": "double_value",
+        "command": "double_value"
+    }
+)
+```
+1. Defines a transformation function that doubles input values.
+2. Specifies input and output schemas for type safety.
+3. Creates a simple input table with test data.
+4. Creates the UDXF expression with the transformation function.
+
+### Build the UDXF expression
+
+Build the expression:
+
+```bash
+xorq build udxf_example.py --expr-name expr
+```
+
+This creates a second `<HASH>` in your `builds/<HASH>/` directory with your serialized UDXF expression. 
+
+![](../../images/quickstart/second-hash.png)
+
+### Start the Flight server
+
+Start the Flight server with your built UDXF expression. Replace `<BUILD_HASH>` with the second hash from your build output:
+
+```bash
+xorq serve-flight-udxf --port 8001 builds/<BUILD_HASH>
+```
+
+Your UDXF is now running as an endpoint on `localhost:8001`. Keep this terminal window open while you query it.
+
+### Query your served UDXF
+
+With the server running, create a new Python file called `query_udxf.py`:
```

Comment:
The query_udxf.py should be updated to query the actual udxf being served. This is still thinking of a penguins udxf being served